src/utils/index.ts

```diff
@@ -20,10 +22,21 @@ export const getValueFromLocalStorage = <T>(key: string): T | null => {
   try {
     const value = localStorage.getItem(key);
 
-    return value as T;
+    if (value !== null) {
+      return JSON.parse(value) as T;
+    }
+
+    return null;
   } catch (error) {
     console.error('Error retrieving value from local storage:', error);
+    toast.error('Error retrieving value from local storage');
```

Comment:
I am not sure that real users need and expect this kind of technical error. It's better to remove this line.

Reply:
Okay!